include/ipc_service/rpmsg_service.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_IPC_SERVICE_RPMSG_SERVICE_H_
+#define ZEPHYR_INCLUDE_IPC_SERVICE_RPMSG_SERVICE_H_
+
+#include <openamp/open_amp.h>
```

Comment:
I would leave it out of the scope of this PR. If anyone is interested in making it even more generic, I think it can be refactored and addressed separately